pkg/apis/troubleshoot/v1beta2/hostanalyzer_shared.go

```diff
@@ -1,13 +1,15 @@
 package v1beta2
 
 type CPUAnalyze struct {
-	AnalyzeMeta `json:",inline" yaml:",inline"`
-	Outcomes    []*Outcome `json:"outcomes" yaml:"outcomes"`
+	AnalyzeMeta   `json:",inline" yaml:",inline"`
+	CollectorName string     `json:"collectorName,omitempty" yaml:"collectorName,omitempty"`
```

Comment:
This change is a little unexpected.  How did this work without collector name?  Or is this a new dependency because of path changes?

Reply:
looks like it was only defined for the analyzers that actually this field, and now these analyzers are also updated to do that same.

Reply:
Yeah, i was conflicted on this one honestly. Anywhere I added `CollectorName` it still isn't used, but was just thinking it was better to make all of the specs consistent since we usually document `CollectorMeta` and `AnalyzeMeta` as being default.

The collectors/analyzers where `CollectorName` was previously omitted are ones like `cpu`, `time`, `memory`, etc. These all will return the same result no matter how many times you run them so no point in specifying them multiple times hence needing to distinguish them by the `CollectorName`